config/api_server_deployment.yml

```diff
@@ -68,10 +68,10 @@ spec:
           resources:
             requests:
               cpu: 100m
-              memory: 300Mi
+              memory: 250Mi
```

Comment:
The local worker doesn't do much in cf-for-k8s. On CF for VMs it is ferrying large files to the blobstore, but in cf-for-k8s it's just moving a file on disk to a mount that the `registry-buddy` has access to. I've lowered both the limit and requests to account for this.